05/app.js

```diff
@@ -0,0 +1,59 @@
+
+document.addEventListener('DOMContentLoaded', function () {
+    const form = document.querySelector('form');
+    form.setAttribute('novalidate', true);
+
+    form.addEventListener('submit', function (event) {
+        event.preventDefault(); 
+
+        const firstName = form.elements.firstName.value;
+        const lastName = form.elements.lastName.value;
+        const street = form.elements.street.value;
+        const houseNumber = form.elements.houseNumber.value;
+        const flatNumber = form.elements.flatNumber.value;
+        const zip = form.elements.zip.value;
+        const city = form.elements.city.value;
+        const voivodeship = form.elements.voivodeship.value;
+
+        const errors = [];
+        const messagesContainer = document.querySelector('.messages');
+        messagesContainer.innerHTML = ''; 
+
+        if (firstName.length === 0) errors.push('Uzupełnij pole "Imię".');
+        if (lastName.length === 0) errors.push('Uzupełnij pole "Nazwisko".');
+        if (street.length === 0) errors.push('Uzupełnij pole "Ulica".');
+        if (city.length === 0) errors.push('Uzupełnij pole "Miejscowość".');
```

Comment:
Sporo tutaj tych if-ów, można zrobić to trochę inaczej - tutaj znajdziesz więcej informacji: 
https://devmentor.pl/b/walidacja-formularza-w-javascript

PS. To tak na przyszłość - nie musisz teraz tego modyfikować.